.github/workflows/actions.yml

```diff
@@ -32,8 +32,13 @@ jobs:
           ${{ runner.os }}-pip-
     - name: Install dependencies
       run: |
-        pip install tensorflow==2.10.0
+        pip install tensorflow-cpu==2.10.0
```

Comment:
why do we have to pin this specifically?

This doesn't break our GPU compatibility - right?